backend/.env

```diff
@@ -21,22 +21,28 @@ TOURNAMENTS_REGISTRATIONS_DATABASE=MONGO
 #TOURNAMENTS_REGISTRATIONS_DATABASE=INMEMORY
 
 PLAYERS_MATCHING_MODULE=ENABLED
-PLAYERS_MATCHING_MODULE_DATABASE=INMEMORY
+PLAYERS_MATCHING_DATABASE=MONGO
```

Comment:
Chcemy miec mongo by default? 

Można też by default INMEMORY, zeby mozna bylo sobie odpalic latwo apke. 
A kazdy w .env.local sobie ustawi co chce aktualnie do developowania.

Reply:
Jak ustawisz:
MONGO_REPOSITORIES=DISABLED
#MONGO_REPOSITORIES=ENABLE
to i tak powinno pójść na InMemory